test/models/backlink.test.js

```diff
@@ -1,42 +1,58 @@
 const faker = require('faker')
-const utils = require('../utils.js')
 
 describe('Backlink', () => {
-  const Backlink = utils.models.Backlink
-  const Page = utils.models.Page
-  const Revision = utils.models.Revision
-  const conn = utils.mongoose.connection
-  const appUrl = 'http://localhost:3000'
+  let Backlink
+  let Page
+  let Revision
+  let conn
   let user
 
-  beforeAll(async () => {
+  beforeAll(done => {
+    Backlink = crowi.model('Backlink')
+    Page = crowi.model('Page')
+    Revision = crowi.model('Revision')
+    conn = crowi.getMongo().connection
+
+    done()
+  })
+
+  beforeAll(async done => {
```

Comment:
It's also unnecessary. I think `done()` should be used only for processing in asynchronously using a callback.